src/Aspire.ProjectTemplates/templates/aspire-servicedefaults/8.2/Aspire.ServiceDefaults1.csproj

```diff
@@ -17,6 +17,7 @@
     <PackageReference Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.9.0" />
     <PackageReference Include="OpenTelemetry.Instrumentation.Http" Version="1.9.0" />
     <PackageReference Include="OpenTelemetry.Instrumentation.Runtime" Version="1.9.0" />
+    <PackageReference Include="AspNetCore.HealthChecks.ApplicationStatus" Version="9.0.0" />
```

Comment:
You can revert changes to the 8.2 templates. We don't support 8.x anymore so they're only there for convenience.